packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -332,6 +337,12 @@ export class OdspDocumentStorageService extends OdspDocumentStorageServiceBase {
 								);
 							}
 						}
+						if (verboseLoggingEnabled) {
+							this.logger.sendTelemetryEvent({
+								eventName: "ObtainSnapshotExtraTime",
+								duration: cacheLookUpTime,
+							});
```

Comment:
instead of new event and verbose logging, can we put thus as a property on event on line 354?

Reply:
Also it looks a big weird from structural POV. I.e. block of code lines 290-323 would report essentially zero time with event (even though we do consult with cache and may get snapshot from cache), while block of code lines 321-338 would report actual time to fetch from cache

Reply:
Removed the other things. Here I just want to look cache time in serial case which is used in our tests.